modules/cnf-topology-aware-lifecycle-manager-platform-update.adoc

```diff
@@ -57,8 +57,6 @@ spec:
       policyName: "platform-upgrade-prep"
```

Comment:
This is no longer needed in the prep policy.

Reply:
@jc-rh This is what you mean? [Diff only](https://github.com/openshift/openshift-docs/pull/65327/commits/4aa4ecaffc001a734d3a082af55a32276fc5fa8d)

Reply:
Yes